README.md

```diff
@@ -1,12 +1,18 @@
 # MeCab.jl
 
-[![Build Status](https://travis-ci.org/chezou/MeCab.jl.svg?branch=master)](https://travis-ci.org/chezou/MeCab.jl)
-[![Coverage Status](https://coveralls.io/repos/chezou/MeCab.jl/badge.svg?branch=master&service=github)](https://coveralls.io/github/chezou/MeCab.jl?branch=master)
-[![MeCab](http://pkg.julialang.org/badges/MeCab_0.3.svg)](http://pkg.julialang.org/?pkg=MeCab&ver=0.3)
-[![MeCab](http://pkg.julialang.org/badges/MeCab_0.4.svg)](http://pkg.julialang.org/?pkg=MeCab&ver=0.4)
+[![Build Status](https://travis-ci.org/snthot/MeCab.jl.svg?branch=master)](https://travis-ci.org/snthot/MeCab.jl)
```

Comment:
Is there any reason to modify these lines?

I think "Build Status" badge should refer to `chezou/MeCab.jl` and package badges should be changed to 0.7 and 1.0. Coverage Status also should keep in the README.

Reply:
Oh I see,let me try to fix

Reply:
I returned to the original text, 
added badges 0.7 and 1.0 but It seems doesn't work now.
Please feel free to modified.

Reply:
No worries. It could be resolved after publishing this package.